### ISSUE TITLE
Ability to run individual specs

### ISSUE THREAD

**Issue by RichardWarburton** [RichardWarburton/lambda-behave#3]
Currently can only run a whole class at a time in IDEs, due to the way the junit runner interacts. This probably needs plugins. In priority order:

intellij
maven
eclipse

**Comment by SveTob** [comment]
After trying lambda-behave for a while I like it a lot. This feature, is the only thing I am missing before pushing others to use this, as we do have big time-consuming tests that we don't want to run when we are just debugging one small individual test.

**Comment by RichardWarburton** [comment]
I agree its an important feature, but tbh I'm hoping that the junit-lambda project will kick off the ground properly so I don't have to write custom plugins for all the IDEs and build systems! Lambda-behave could possibly just fit into their more sophisticated model if they meet all the requirements.

**Comment by jypma** [comment]
Junit-lambda seems to be fairly far away. In the meantime, would it make sense to have a kind of `only()` method or sth, which one could toss into the test code of the only spec to be run? That way you can at least temporarily, while developing, only run a single spec from a class. You'd then delete that line again before committing.
